vmalert: enable configuring explicit path (#1536)
VictoriaMetrics/VictoriaMetrics: app/vmalert/README.md

    	Optional TLS server name to use for connections to -remoteWrite.url. By default the server name from -remoteWrite.url is used
  -remoteWrite.url string
    	Optional URL to VictoriaMetrics or vminsert where to persist alerts state and recording rules results in form of timeseries. E.g. http://127.0.0.1:8428
  -remoteWrite.disablePathAppend

>     	Optionally disable appending '/api/v1/write' to the remoteWrite url, enabling configuring an absolute path

Reviewer: ```suggestion
     Whether to disable automatic appending of '/api/v1/write' path to the configured -remoteWrite.url.
```
